.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   env: { browser: true, es2015: true, node: true },
-  settings: { "import/resolver": "webpack" },
+  settings: { "import/resolver": "webpack", react: { version: "detect" } },
```

Comment:
We were missing `react: { version: "detect" }` which is why the why the react warning is always popping up. This allows eslint to auto detect the React version provided by Docusarus.